commons/src/main/java/org/dhis2/commons/orgunitselector/OUTreeViewModel.kt

```diff
@@ -53,7 +53,7 @@ class OUTreeViewModel(
     fun searchByName(name: String) {
         if (name.isEmpty()) {
             fetchInitialOrgUnits()
-        } else if (name.length >= 3) {
+        } else if (name.length >= 2) {
```

Comment:
do you think that a test to check if it fetches the full list or filtered list depending on the name length would be valuable?